Examples/FriendlyFlix/README.md

```diff
@@ -0,0 +1,133 @@
+# FriendlyFlix
+
+![FriendlyFlix, a movie tracker sample app](FriendlyFlix-Hero.png)
+
+## Introduction
+
+This quickstart is a movie tracker app to demonstrate the use of Firebase Data Connect
+ with a Cloud SQL database.
+
+For more information about Firebase Data Connect visit [the docs](https://firebase.google.com/docs/data-connect/).
+
+
+## Prerequisites
+
+To use this quickstart, you'll need the following:
+- A computer running macOS
+- The latest version of [Xcode](https://developer.apple.com/xcode/)
+- The latest version of [Visual Studio Code](https://code.visualstudio.com/)
+- The [Firebase Data Connect VS Code Extension](https://marketplace.visualstudio.com/items?itemName=GoogleCloudTools.firebase-dataconnect-vscode)
+- The latest version of the [Firebase CLI](https://firebase.google.com/docs/cli#update-cli)
+
+## Getting Started (local development)
+
+Follow these steps to get up and running with Firebase Data Connect locally (i.e., without setting up a production SQL instance). At the end of this guide, you will find instructions for connecting to a production SQL instance.
+
+For more detailed instructions,
+check out the [official documentation](https://firebase.google.com/docs/data-connect/quickstart-local).
+
+
+### 1. Create a Firebase project
+
+1. If you haven't already, create a Firebase project.
+    * In the [Firebase console](https://console.firebase.google.com), click
+        **Create a project**, then follow the on-screen instructions.
+
+### 2. Get the code for the FriendlyFlix quickstart app
+
+1. Clone this repository to your local machine:
+   ```sh
+   git clone https://github.com/firebase/data-connect-ios-sdk
+   ```
+
+### 3. Open in Visual Studio Code (VS Code)
+
+1. Open the `data-connect-ios-sdk/Examples/FriendlyFlix` directory in VS Code.
+2. Click on the **Firebase Data Connect** icon on the VS Code sidebar to load the Firebase Data Connect Extension.
+   a. Sign in with your Google Account if you haven't already.
+3. Click on **Connect a Firebase project** and choose the project you created in the first step.
+4. Click on **Start emulators**.
+
+### 4. Populate the database
+In VS Code, open the `data-connect-ios-sdk/Examples/FriendlyFlix/dataconnect/data_seed.gql` file and click the
+ **Run (local)** button at the top of the file.
+
+If you'd like to confirm that the data was correctly inserted,
+open `data-connect-ios-sdk/Examples/FriendlyFlix/dataconnect/movie-connector/queries.gql` and run the `ListMovies` query.
+
+### 5. Run the app
+
+1. Open `data-connect-ios-sdk/Examples/FriendlyFlix/app/FriendlyFlix/FriendlyFlix.xcodeproj` in Xcode
+2. Wait for all packages to be resolved.
+3. Select one of the iPhone Simulators as the run destination.
+4. Press the **Run** button in Xcode to run the sample app on the iOS Simulator.
+
+> [!Note] If you've used Firebase before, you might be wondering why you didn't have to download the
```

Comment:
Are you sure about this? I think the `FirebaseApp.configure()` call requires a local plist file with an app configuration i.e. you do need to define an iOS app in the Console.

Reply:
I added a plist file that contains dummy values, which is why you don't have to download one in the first section of the README walkthrough.

Reply:
https://github.com/firebase/data-connect-ios-sdk/pull/21/files#diff-f4bf96b05ebcfdfde75ec516f2a94bb02ea16b0c12d380814a60d08055ab9c04